src/backend/joanie/tests/core/test_models_order_get_or_generate_certificate_for_credential_product.py

```diff
@@ -186,6 +187,7 @@ def test_models_order_get_or_generate_certificate_for_credential_product_enrollm
             target_courses=[course_run.course],
         )
         order = factories.OrderFactory(product=product)
+        order.submit()
```

Comment:
```suggestion
        order = factories.OrderFactory(product=product, state=ORDER_STATE_VALIDATED)
```

It does not work as product is free ?

Reply:
Submit creates the `OrderTargetCourseRelation` so it needs to be called